### ISSUE TITLE
Question regarding Swift.stg

### ISSUE THREAD

**Issue by hanjoes** [janyou/ANTLR-Swift-Target#4]
hello, i noticed this line:

`AttributeDecl(d) ::= "<d.name>: <SwiftTypeMap.(d.type)><if(d.initValue)> = <d.initValue><else>!<endif>"
`
in Swift.stg which adds forced unwrap to all attributes that don't have values in context objects. What's the rationale behind this? Cases are that if user wants to use ordinary optional it will add an extra "!" to the attribute, e.g.: public var name: Type?!

**Comment by janyou** [comment]
should be just to simplify.

**Comment by hanjoes** [comment]
cool, i'm working on a tiny patch to support variable decl idiom in languages like swift and ran into the force unwrap issue above, just thinking it might be better that we leave out the default force unwrap in Swift.stg and let user decide whether the attribute should be an optional :)

**Comment by janyou** [comment]
you are right. i think it may affect test template files(in org/antlr/v4/test/runtime/templates),you can try it.

**Comment by hanjoes** [comment]
k, I will take a look.
